lib/ttt_app.rb

```diff
@@ -0,0 +1,11 @@
+module TttApp
+  class Error < StandardError; end
+  # Your code goes here...
+  print('Welcom To TTT')
+end
+
+class TicTacToe
+  def self.printWelcome
+      'Welcome to Tic Tac Toe Game'
+  end
+end
```

Comment:
By convention, you want to include class within a module. The module name is often a folder and serves as a namespace that helps you organize your code. For example,

```rb
# lib/ttt_app/error.rb

module TttApp
  class Error < StandardError; end
end
```

```rb
# lib/ttt_app/tic_tac_toe.rb

module TttApp
  class TicTacToe
    def self.print_welcome
      puts 'Welcome to Tic Tac Toe Game'
    end
  end
end
```

Reply:
Thank. I will do that now.